internal/handlers/tigris/tigrisdb/databases.go

```diff
@@ -43,7 +43,7 @@ func (tdb *TigrisDB) createDatabaseIfNotExists(ctx context.Context, db string) (
 	// If we detect that other creation is in flight, we give up to three attempts to create the database.
 	// TODO https://github.com/FerretDB/FerretDB/issues/1341
 	for i := 0; i < 3; i++ {
-		err = tdb.Driver.CreateDatabase(ctx, db)
+		_, err = tdb.Driver.CreateProject(ctx, db)
```

Comment:
Let's make sure I understand this, if db exists `CreateProject` won't return error.
If db does not exist `DeleteProject` won't return error.
For our usage, we always want to use `Project` instead of `Database`. Let me know if I'm wrong.

Reply:
Basically, "project" is a combination of database and something else we don't need.
`CreateProject` seems to work the same way as `CreateDatabase` in all cases.

Reply:
sounds good :+1: